tests/testthat/test-prepare_istr_company.R

```diff
@@ -1,23 +1,46 @@
 test_that("total number of rows for a comapny is either 1 or 3", {
-  company <- unnest_company(toy_sector_profile_upstream_output())
-  product <- unnest_product(toy_sector_profile_upstream_output())
+  local_options(readr.show_col_types = FALSE)
+
+  companies <- read_csv(toy_sector_profile_upstream_companies())
+  scenarios <- read_csv(toy_sector_profile_any_scenarios())
+  inputs <- read_csv(toy_sector_profile_upstream_products())
+  .inputs <- add_rowid(inputs)
+  output <- sector_profile_upstream(companies, scenarios, .inputs)
+  .product <- unnest_product(output)
+
+  y <- select(.inputs, matches(extra_cols_pattern()))
+  x <- select(.product, -any_of(names(y)), extra_rowid())
+  product <- left_join(x, y, relationship = "many-to-many", by = extra_rowid())
+  company <- unnest_company(output)
 
   out <- prepare_istr_company(
     company,
     product,
     ep_companies,
     ecoinvent_activities,
     small_matches_mapper,
-    ecoinvent_inputs
+    ecoinvent_inputs,
+    isic_tilt_mapper
   ) |>
     group_by(companies_id, scenario, year) |>
     summarise(count = n())
   expect_true(all(unique(out$count) %in% c(1, 3)))
 })
 
 test_that("handles tiltIndicator output", {
-  company <- unnest_company(toy_sector_profile_upstream_output())
-  product <- unnest_product(toy_sector_profile_upstream_output())
+  local_options(readr.show_col_types = FALSE)
+
+  companies <- read_csv(toy_sector_profile_upstream_companies())
+  scenarios <- read_csv(toy_sector_profile_any_scenarios())
+  inputs <- read_csv(toy_sector_profile_upstream_products())
+  .inputs <- add_rowid(inputs)
+  output <- sector_profile_upstream(companies, scenarios, .inputs)
+  .product <- unnest_product(output)
+
+  y <- select(.inputs, matches(extra_cols_pattern()))
+  x <- select(.product, -any_of(names(y)), extra_rowid())
+  product <- left_join(x, y, relationship = "many-to-many", by = extra_rowid())
+  company <- unnest_company(output)
```

Comment:
@maurolepore To run the old tests, I had to write extra code which needs to be fixed using correct toy datasets. (Flagging)

Reply:
See my previous comment about hacking `toy_sector_profile_upstream_output()` to add the column you want and point to the relevant issue/PR in tiltToyData. The one change in `toy_sector_profile_upstream_output()` would avoid many many changes in this PR.